### DISCUSSION TITLE
Add Loader in store_manager

### DISCUSSION THREAD

**Discussion by iabhiyaan** [GrapesJS/grapesjs#3650] [Q&A] [answered]
Hello @artf, How can we add loader in storage_manager while loading html from api ?
```js
  editor.on('storage:end:load', () => {
    document.querySelector('#editor-loader').classList.add('hide');
  });
```
This kinda works! html is rendered but becomes inactive for more than 4-5 secs.
Can we wait until canvas is ready to play ?

**Comment by artf** [comment, marked as answer]
Do it on `load`, which is triggered after the storage load and when the canvas is ready to be interactive

**Reply by iabhiyaan** [reply]
Thanks @artf
